### DISCUSSION TITLE
there is something to say about Black,

### DISCUSSION THREAD

**Discussion by ElCuboNegro** [johnthagen/python-blueprint#60] [Ideas]
It could sounds something dumb because is easy fixable, but the meta information of black gives a lot of headches when you try to use it with Conda. the problem? they don't follow the double quote standard in their meta information. I know, is dumb but a great why to choose other lints. 

by the way, have you though about give examples of integration with tools like sonarlint/sonarcube?

**Comment by johnthagen** [comment]
> It could sounds something dumb because is easy fixable, but the meta information of black gives a lot of headches when you try to use it with Conda. the problem? they don't follow the double quote standard in their meta information. I know, is dumb but a great why to choose other lints.

Could you provide some more information about this? Are you talking about Conda's config file or `black`'s `pyproject.toml`? I haven't used Conda much and I'd like to keep `python-blueprint` a pure-Python example that doesn't require any extra tooling other than can be bootstrapped with what comes with CPython (`venv`, `pip`, etc.).

In any case, `python-blueprint` is just an example so people are free to pick and choose which pieces they find useful. It's expected some users may not want to use `black`, I myself held off for many years.

> by the way, have you though about give examples of integration with tools like sonarlint/sonarcube?

I haven't used these tools before.
